src/analysis/ConfigurationExt.hpp

```diff
@@ -1,5 +1,7 @@
 /*
-  Copyright (C) 2012-2015
+  Copyright (C) 2016
+      Max Planck Institute for Polymer Research & Johannes Gutenberg-Universität Mainz
+  Copyright (C) 2012,2013
```

Comment:
I though about adding it, but @govarguz suggested me to put only the year of the commit I'm doing (2016 in this case). Thoughts on that? Let me know and I will do the changes if I need to add for example in this case: 2014,2015,2016.
I guess the same applies for example to src/analysis/ConfigurationsExt.cpp just to name one.